core/web/external_initiators_test.go

```diff
@@ -0,0 +1,179 @@
+package web_test
+
+import (
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	"github.com/smartcontractkit/chainlink/core/internal/cltest"
+	"github.com/smartcontractkit/chainlink/core/store/models"
+	"github.com/smartcontractkit/chainlink/core/web"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNotifyExternalInitiator_Notified(t *testing.T) {
+	tests := []struct {
+		Name          string
+		ExInitr       models.ExternalInitiatorRequest
+		JobSpec       models.JobSpec
```

Comment:
Thanks for the note.  My personal preference is to keep it as is because the parts of the struct which affect the test is clear.  I'm still happy to go for using testdata for consistency reasons if that's a factor? or maybe I'm missing some other important point?

Reply:
I don't think we're consistent enough to call it a convention, so it really just comes down to personal preference.